.gitignore

```diff
@@ -1,16 +1,8 @@
-*.pyc
-*.pyo
-*~
+*.egg-info
+*.py[co]
 *.sw[po]
+*~
 .DS_Store
-.coverage
-.installed.cfg
-MANIFEST
```

Comment:
Not sure about this one. It's here because it's generated by setup sdist, I think